widgets/gui/qt_left_window.py

```diff
@@ -63,7 +66,21 @@ def __init__(self, parent):
                                [self, "stripe_reduction_offset", -10, 10, float, 0.01, 0.58]]
 
         for i in self.parameter_list:
-            self.layout.addWidget(qt_textbox_and_slider.TextboxAndSlider(*i))
+            textbox_and_slider = qt_textbox_and_slider.TextboxAndSlider(*i)
+            self.layout.addWidget(textbox_and_slider)
             self.layout.addWidget(qt_line_break.LineBreak(Qt.AlignTop))
 
+            self.parameter_vals.append(textbox_and_slider)
+
         self.setLayout(self.layout)
+
+        # not in property for testing purposes
+        for i in range(0, 14):
+            for QSpinBox in self.parameter_vals[i].layout.itemAt(1).children():
```

Comment:
why trying to reach from layout?  TextboxAndSlider instances they all have nicely named attributes. Why don't you do `self.parameter_vals[i].spinbox`

Reply:
This is good, I can easily access the right object. I still get attribute error: AttributeError: 'QDoubleSpinBox' object has no attribute 'Value' as before though.

Reply:
The same happens if I call .Value()

Reply:
Nevermind, the v in value needs to be small. Was capitalized in documentation I was using...

Reply:
still trying to access widgets through layouts is not the preferable way. It can look cleaner if you access through attributes as I mentioned.

Reply:
yes, I access it with .spinbox.value()